navigation-test/navigation_test.js

```diff
@@ -0,0 +1,45 @@
+var file_path = process.env.FILE_PATH
+var webdriver = require('selenium-webdriver');
+var assert = require('chai').assert;
+var { Builder, By } = require("selenium-webdriver");
+
+
+
+describe("Testing HtmlForms", function() {
+    this.timeout(3 * 1000 * 60);
+    var driver = new webdriver.Builder().forBrowser('chrome').build()
+
+    before(function setupWebdriver(done) {
+        var file_url = `file:///Users/user/Downloads/Dufuna-CodeCamp20/submissions/oyetejuOloyede/htmlForms/contact.html`;
+       
+        driver.get(file_url).then(done)
+    })
+
+    after(function() {
+        driver.quit()
+    })
+
+    it("test case: check if Contact Us link is present",  async () => {
+        const element = await driver.findElement(By.css("a[href='../htmlForms/contact.html']"));
+        const link = await element.getText()
+        assert.include(link, "Contact Us", `Ensure Link is present`)
+    })
+
+    it("tests case: Check if Product Link is present", async () => {
+        const element = await driver.findElement(By.css("a[href='../introductionToHtml/e-commerceStore.Html']"));
```

Comment:
This is not the common link for most scholars. I think you should update the link here. They should have a product.html file in the HTML folder. This particular scholar you used isn't doing the right thing.